.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
     "files.exclude": {
-        "frontend": true
+        "frontend": false
```

Comment:
Please revert this change. It's in the docs to open the workspace file.

https://github.com/sirgwain/craig-stars/blob/develop/docs/development.md#visual-studio-code

Reply:
k